config/software/libzmq.rb

```diff
@@ -18,7 +18,7 @@
 name "libzmq"
 default_version "2.1.11"
 
-license "GPL-3.0"
+license "GPL-3.0 (with exception)"
```

Comment:
FYI only one other project has this license currently: 
https://github.com/chef/omnibus-software/blob/e8124b356fed937a6e646cdd194f9eaa06f325e6/config/software/config_guess.rb#L23